packages/modules/modules.js

```diff
@@ -0,0 +1,27 @@
+meteorInstall = makeInstaller();
+
+meteorInstall({
+  node_modules: {
+    modules: {
+      "css.js": function (r, exports) {
+        exports.addStyles = function (css) {
+          var doc = document;
+          var style = doc.createElement("style");
+
+          style.setAttribute("type", "text/css");
+
+          if (style.styleSheet) {
```

Comment:
Can we add a comment explaining this if clause? It is a browser compatibility thing?